armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -138,6 +142,14 @@ def performPrescribedAxialExpansion(
     ):
         """Perform axial expansion of an assembly given prescribed expansion percentages.
 
+        .. impl:: Perform expansion/contraction, given a list of components and expansion coefficients.
+            :id: I_ARMI_AXIAL_EXP_PRESC
+            :implements: R_ARMI_AXIAL_EXP_PRESC
+
+        .. impl:: Perform thermal expansion/contraction, given an axial temp distribution over an assembly.
```

Comment:
This needs to get moved down to the `performThermalAxialExpansion` method.

Reply:
Done